### DISCUSSION TITLE
thingiverse adbock

### DISCUSSION THREAD

**Discussion by yasars** [AdguardTeam/AdGuardHome#4331] [Q&A]
look pls here: https://github.com/AdguardTeam/AdguardFilters/issues/111111

@Alex-302 https://github.com/AdguardTeam/AdguardFilters/commit/9c4875e9b5cfec4d92a584ee12a8c1fb6498e9e9

had fix here in adguard filters , thingsverse

how can i fix this on adguard home??  

@@||www3.doubleclick.net^$xmlhttprequest,$other,$domain=thingiverse.com
thingiverse.com#%#//scriptlet('prevent-fetch', 'www3.doubleclick.net')

is doesnt work.... thank you !! thank you @Alex-302

**Comment by yasars** [comment]
found a solution:

disable all not stock adblock lists and add https://raw.githubusercontent.com/AdguardTeam/AdguardFilters/master/EnglishFilter/sections/antiadblock.txt

after that install only stock lists... Easylist,AdGuard DNS filter,stevenblack and AdAway Default Blocklist  and Voilla it works :)

**Comment by Alex-302** [comment]
Add www3.doubleclick.net to exclusions.
